src/ColorPicker/Halogen/Utils/Drag.purs

```diff
@@ -0,0 +1,205 @@
+module ColorPicker.Halogen.Utils.Drag
```

Comment:
BTW, shouldn't we make new lib with drag utils (or perhaps we already have one) ?